script.js

```diff
@@ -1,12 +1,118 @@
 //You can edit ALL of the code here
+let tvShows = getAllShows() 
+ console.dir(tvShows)
+let  filteredEpisode ;
+ const header = document.getElementById("header");
+ let selectionBar = document.getElementById("selection-bar");
+ let selectionShowsBar = document.getElementById("selection-shows-bar")
+//const allEpisodes = getAllEpisodes();
+let allEpisodes 
+fetch(`https://api.tvmaze.com/shows/20/episodes`)
+.then(Response => Response.json())
+.then(data =>{
+    allEpisodes = data
+  //console.log(data)
+})
+  .catch(error => console.error("Sorry for the terrible time"))
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+  //console.log(allEpisodes)
+  selection(allEpisodes);
+}
+
+function makeSingleTVShowListing(episode) {
+  const episodeDiv =  document.createElement("div")
+  let banner = document.createElement("h2")
+  banner.innerText = ` ${episode.name} - S${addingZero(episode.season)}E${addingZero(episode.number)}`;
+  let image = document.createElement("IMG")
+   episodeDiv.className = "episode"
+   episodeDiv.appendChild(banner)
+  image.src = episode.image.medium
+  episodeDiv.appendChild(image)
+  let plot = document.createElement("div")
+  plot.innerHTML = episode.summary
+  episodeDiv.appendChild(plot)
+  return episodeDiv;
 }
+// adding the zero function
+ function addingZero (num){
```

Comment:
nice function . you could use padStart() as well.

Reply:
hi, would you be able to slack me documentation regarding the padStary()
thanks for giving me time to review my work